packages/shared/src/models/bootc.ts

```diff
@@ -29,3 +29,5 @@ export interface BootcBuildInfo {
 }
 
 export type BootcBuildStatus = 'running' | 'creating' | 'success' | 'error' | 'lost' | 'deleting';
+
+export const BootcHistoryFilename = 'history.json';
```

Comment:
I think variables or const should be either all UPPER_CASE or just camelCase here it's more like a Class/Interface/Type